operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/dependent/KubernetesDependentResourceController.java

```diff
@@ -13,14 +13,13 @@
 
 public class KubernetesDependentResourceController<R extends HasMetadata, P extends HasMetadata>
```

Comment:
TBH, don't see why we need this controller per resource model. Why cannot just the dependent resource manager handle this? Like having it  aggregate the controllers and the configuration. But this is probably something to discuss if we decide we got his way.

Reply:
Because it needs to be passed around at the moment. This change is about putting together the informer part and the ownership configuration into a single coherent configuration. Cleans up things by showing what belongs together. Helps things with the Quarkus extension as well. 
We might indeed revisit, clean things further later but this change makes sense within the current scope.